### ISSUE TITLE
Support basic http authorization

### ISSUE THREAD

**Issue by aquaminer** [browsh-org/browsh#170]
when you try to load 192.168.1.1 (browsh --startup-url http://192.168.1.1) get an eternal loading page, please add support for http basic authorization

**Comment by tobimensch** [comment]
I think this is a corner case, as classic http authorization is rarely used nowadays. This is why I added the low priority label.

**Comment by aquaminer** [comment]
as far as I understand for the basic http authorization component is used prompt javascript, and Uvas also no support alert, prompt, confirm, which are used much more often

**Comment by tobimensch** [comment]
@aquaminer 
I'm not sure that's true. If so, then your issue would be fixed  along with this one #26 .

**Comment by aquaminer** [comment]
@tobimensch I'm also not entirely sure of my words, but it POPs up just like the comfirm window

**Comment by tombh** [comment]
I think this is actually related to #106 as the dialogue is native to the Browser's UI not the DOM.

**Comment by aquaminer** [comment]
@tombh thank you for explaining, Yes then this applies to #106

**Comment by henroFall** [comment]
For logging into other appliances that use basic auth like webcams, routers, etc

**Comment by runekaagaard** [comment]
Yes I still got several devices and webservers behind basic http authentication so it would be a great feature IMO but probably not simple to code.

As a workaround, this works fine for me:

```bash
read -s PASS
# Enter password and press enter
browsh https://[USERNAME]:$PASS@my.url.com
```
